packages/database/exp/api.ts

```diff
@@ -0,0 +1,77 @@
+/**
+ * @license
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export {
+  FirebaseDatabase,
+  enableLogging,
+  getDatabase,
+  goOffline,
+  goOnline,
+  useDatabaseEmulator,
+  repoManagerDatabaseFromApp as _repoManagerDatabaseFromApp
+} from '../src/exp/Database';
+export {
+  Query,
+  Reference,
+  ListenOptions,
+  Unsubscribe,
+  ThenableReference
+} from '../src/exp/Reference';
+export { OnDisconnect } from '../src/exp/OnDisconnect';
+export {
+  DataSnapshot,
+  EventType,
+  QueryConstraint,
+  QueryConstraintType,
+  endAt,
+  endBefore,
+  equalTo,
+  get,
+  limitToFirst,
+  limitToLast,
+  off,
+  onChildAdded,
+  onChildChanged,
+  onChildMoved,
+  onChildRemoved,
+  onDisconnect,
+  onValue,
+  orderByChild,
+  orderByKey,
+  orderByPriority,
+  orderByValue,
+  push,
+  query,
+  ref,
+  refFromURL,
+  remove,
+  set,
+  setPriority,
+  setWithPriority,
+  startAfter,
+  startAt,
+  update,
+  ReferenceImpl as _ReferenceImpl,
+  QueryImpl as _QueryImpl,
+  child as _child
```

Comment:
`child` should be part of the public API. We can just export it as child (see go/rtdb-next)

Reply:
Thanks. Updated.